### ISSUE TITLE
Some packages have NULL versions

### ISSUE THREAD

**Issue by wch** [metacran/crandb#43]
For example:
http://crandb.r-pkg.org/subplex/all
http://crandb.r-pkg.org/lme/all

**Comment by gaborcsardi** [comment]
Yeah, these have free text version numbers, like `3.0b8a-2 (1999/06/07)` and I am not sure how to parse and sort them.

**Comment by maelle** [comment]
no way to keep the original string?

**Comment by gaborcsardi** [comment]
we rely on being able to order them, so this would require some work

**Comment by maelle** [comment]
what I don't understand, because I haven't looked at the source enough, is why one can't just order by date?

**Comment by gaborcsardi** [comment]
dates are often hard to come by for old packages.
